charts/kube-prometheus-stack/values.yaml

```
@@ -4065,7 +4065,7 @@ prometheus:
     ## Defines the maximum time that the `prometheus` container's startup probe
     ## will wait before being considered failed. The startup probe will return
     ## success after the WAL replay is complete. If set, the value should be
-    ## greater than 60 (seconds). Otherwise it will be equal to 600 seconds (15
+    ## greater than 60 (seconds). Otherwise it will be equal to 600 seconds (10
```

According to @jkroepke last comment this is what should be fixed

```suggestion
    ## greater than 60 (seconds). Otherwise it will be equal to 900 seconds (15
```